sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/ObjectsShouldNotBeDisposedMoreThanOnce.cs

```diff
@@ -65,67 +64,20 @@ public void Disposed_Using_WithExpressions()
             }
         }
 
-        public void Disposed_Using3(Stream str)
```

Comment:
I suggest to leave these tests and mark them as compliant.

Reply:
As discussed I've added back 2 test cases with streams